src/Tests/WebApp.fs

```diff
@@ -323,7 +323,9 @@ let tests = testList "Web App Tests" [
             |> getResource<Arm.Web.Site>
             |> List.filter (fun x -> x.ResourceType = Arm.Web.slots)
         // Default "production" slot is not included as it is created automatically in Azure
-        Expect.hasLength slots 1 "Should only be 1 slot"
+

```

Comment:
We should add the zip_deploy to the site to prove we're actually removing it rather than just not setting it.

Reply:
Ah good spot - have updated